e2e-tests/tests/test_anonymizer.py

```diff
@@ -109,7 +109,7 @@ def test_given_anonymizers_called_then_expected_anonymizers_list_returned():
     response_status, response_content = anonymizers()
 
     expected_response = """
-        ["hash", "mask", "redact", "replace", "encrypt"]
+        ["hash", "mask", "redact", "replace", "encrypt", "replace", "custom"]
```

Comment:
I would leave custom out here, as its operational with package usage, not rest api. Maybe can be controlled with the operator_name interface method.

Reply:
yes, still in draft. this functionality is only for the package. WIP